config/seed/fixtures/posts.yml

```diff
@@ -338,6 +324,262 @@
+  updated_at: <%= Time.now %>
+  published_at: <%= Time.now %>
+  page: false
+  featured: false
+  language: en_US
+  status: published
+  slug: beneficio-2
+  title: Infraestructura en la nube (a travéz de socios tecnológicos con IBM Y Microsoft)
+  markdown: <%= fake_sentence %>
+
+- uuid: <%= uuid %>
+  created_at: <%= Time.now %>
+  updated_at: <%= Time.now %>
+  published_at: <%= Time.now %>
+  page: false
+  featured: false
+  language: en_US
+  status: published
+  slug: beneficio-3
+  title: Acceso a laboratorios de prototipado, pasantías de estudiantes y otros servicios previstos por UTPL
+  markdown: <%= fake_sentence %>
+
+# Coworking: Paquetes 
+- uuid: <%= uuid %>
+  created_at: <%= Time.now %>
+  updated_at: <%= Time.now %>
+  published_at: <%= Time.now %>
+  page: false
+  featured: false
+  language: en_US
+  status: published
+  slug: coworking-paquetes
+  title: Paquetes para contratar
+  markdown: <%= fake_sentence %>
+
+# Coworking: Paquetes carrusel
+- uuid: <%= uuid %>
+  created_at: <%= Time.now %>
+  updated_at: <%= Time.now %>
+  published_at: <%= Time.now %>
+  page: false
+  featured: false
+  language: en_US
+  status: published
+  slug: paquete-1
+  title: Paquete Villonaco
+  meta_title: $65
+  markdown: |
+    - Espacio físico para una persona
+    - Servicios básicos
+    - Internet
+    - Taller  o conferencia mensual
+    - Casillero
+    - Equipo de computo
+    - Sala de runiones (4 horas a mes)
+    - Infraestructura en la nube
+    - Acceso al laboratorio de prototipo, pasantías de estudiantes en los emprendimientos, y otro servicios porvistos por UTLP
+
+- uuid: <%= uuid %>
+  created_at: <%= Time.now %>
+  updated_at: <%= Time.now %>
+  published_at: <%= Time.now %>
+  page: false
+  featured: false
+  language: en_US
+  status: published
+  slug: paquete-2
+  title: Paquete Opcional 
+  meta_title: $50
+  markdown: |
+    - Espacio físico para una persona
+    - Servicios básicos
+    - Internet
+    - Taller  o conferencia mensual
+    - Casillero
+    - Equipo de computo
+    
+# Coworking: Galeria
+- uuid: <%= uuid %>
+  created_at: <%= Time.now %>
+  updated_at: <%= Time.now %>
+  published_at: <%= Time.now %>
+  page: false
+  featured: false
+  language: en_US
+  status: published
+  slug: coworking-galeria
+  title: Galeria
+  markdown: <%= fake_sentence %>
+  
+# Coworking: Galeria carrusel
+- uuid: <%= uuid %>
+  created_at: <%= Time.now %>
+  updated_at: <%= Time.now %>
+  published_at: <%= Time.now %>
+  page: false
+  featured: false
+  language: en_US
+  status: published
+  slug: galeria-1
+  title: Galeria 1
+  markdown: <%= fake_sentence %>
+  image: /default/logo_conectate_color.png
+
+- uuid: <%= uuid %>
+  created_at: <%= Time.now %>
+  updated_at: <%= Time.now %>
+  published_at: <%= Time.now %>
+  page: false
+  featured: false
+  language: en_US
+  status: published
+  slug: galeria-2
+  title: Galeria 2
+  markdown: <%= fake_sentence %>
+  image: /default/logo_noggalito_color.png
```

Comment:
a estas imágenes hay que reemplazarlas verdad?

ej `http://placehold.it/350x150?text=imagen+1`

![](http://placehold.it/350x150?text=imagen+1)